examples/zcap_invocation.jsonld

```diff
@@ -0,0 +1,12 @@
+{
+    "@context": "https://w3id.org/security/v2",
+ "id": "urn:uuid:ad86cb2c-e9db-434a-beae-71b82120a8a4",
+ "capabilityAction": "Drive",
+ "proof": {
```

Comment:
There is a whitespace inconsistency here.

It would be nice if the example files were usable/verifiable. But testing the parsing is good too.